zeebe/exporters/elasticsearch-exporter/src/main/java/io/camunda/zeebe/exporter/ElasticsearchClient.java

```diff
@@ -175,6 +175,17 @@ public boolean putIndexTemplate(final ValueType valueType) {
     return putIndexTemplate(templateName, template);
   }
 
+  public boolean putIndexTemplate(final ValueType valueType, final String version) {
+    final String templateName = indexRouter.indexPrefixForValueType(valueType, version);
+    final Template template =
+        templateReader.readIndexTemplate(
+            valueType,
+            indexRouter.searchPatternForValueType(valueType, version),
+            indexRouter.aliasNameForValueType(valueType));
+
+    return putIndexTemplate(templateName, template);
+  }
```

Comment:
🔧 You can reduce some duplication here by changing the method above this to just call this one with the `VersionUtil.getVersionLowerCase()` as the version

Reply:
yep was just doing it quick and dirty to see if it was worth doing